config/default.yml

```diff
@@ -3660,6 +3660,12 @@ Style/CollectionMethods:
     - inject
     - reduce
 
+Style/CollectionQuerying:
```

Comment:
Naming is hard... I named the cop like this because of the cop [`Style/CollectionMethods`](https://docs.rubocop.org/rubocop/cops_style.html#stylecollectionmethods) which is also related to `Enumerable`, and because in Ruby docs, replacement methods are called [Methods for Querying](https://docs.ruby-lang.org/en/3.4/Enumerable.html#module-Enumerable-label-Methods+for+Querying).

Reply:
Interesting - I wasn't aware of this and it should probably be mentioned somewhere in the docs and the matching style guide rule.

Reply:
Just fyi, I've opened a PR for the style guide: https://github.com/rubocop/ruby-style-guide/pull/962